app/helpers/order_cycles_helper.rb

```diff
@@ -37,12 +37,10 @@ def permitted_hub_enterprise_options_for(order_cycle)
   end
 
   def distributors_with_editable_shipping_and_payment_methods(order_cycle)
-    return order_cycle.distributors if order_cycle.coordinator
-      .in? Enterprise.managed_by(spree_current_user)
+    return order_cycle.distributors if Enterprise
+      .managed_by(spree_current_user).exists?(order_cycle.coordinator.id)
```

Comment:
I am just trying to understand this, so spelling it out "loud" : 
So if the current user can manage the order cycle's coordinator, we show all the order cycle's distributors, I believe this covers the super admin ? 
If not we just show all the distributors that the current user can manage.

Reply:
Yes @rioug,  that's correct.